PyU4V/tests/ci_tests/test_pyu4v_ci_system.py

```diff
@@ -502,3 +502,93 @@ def test_download_audit_log_record_return_binary(self):
         self.assertTrue(response.get(SUCCESS))
         self.assertIn(BINARY_DATA, response.keys())
         self.assertIsInstance(response.get(BINARY_DATA), bytes)
+
+    def test_get_director_list(self):
+        """Test get_iscsi_director_list."""
```

Comment:
nit 
""" Test get_director_list for iscsi."""  or """Test get_director_list."""
whichever is appropriate